### PR TITLE
Bug 1889213: The error message of uploading failure is not clear enough

### PR DESCRIPTION
**Pull request by glekner**

### CODE REVIEW

**Comment by glekner** [comment]
Screen Shot 2020-12-15 at 23 28 17

Screen Shot 2020-12-15 at 23 28 20

@matthewcarleton @rawagner @jelkosz

**Comment by matthewcarleton** [comment]
nice work! 
Should the action say Back to form (deletes DV)? We don't mention the PVC at all prior to the action so it might be a bit confusing.

**Comment by glekner** [comment]
changed button text to - `(Deletes DV)` @matthewcarleton

File: frontend/packages/kubevirt-plugin/src/k8s/requests/cdi-upload/cdi-upload-requests.ts [hunk 1]
```diff
@@ -65,7 +64,7 @@ const createUploadToken = async (pvcName: string, namespace: string): Promise<st
   }
 };
 
-export const createUploadPVC = async (dataVolume: V1alpha1DataVolume, t: TFunction) => {
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/k8s/requests/cdi-upload/cdi-upload-requests.ts:68]: nice !

**Pull request merged**